styles/styles.css

```diff
@@ -0,0 +1,384 @@
+  margin-right: auto;
+  margin-left: auto;
+}
+
+p.copyright {
+  padding-bottom: 2rem;
+}
+
+/* blog styling */
+.blog-entry {
+    margin: 0 5rem 4rem 2rem;
+    overflow: auto;
+}
+.blog-image {
+    float: left;
+}
+.blog-image img {
+    display: block;
+    margin-right: 1rem;
+}
+.blog-content {
+    margin-left: 1rem;
+    margin-top: 0px;
+}
+.blog-title {
+    font-weight: bold;
+    font-size: 125%;
+    margin-top: 0px;
+}
+
+/* dog styling */
+.all-dogs-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.dog-adoption-card {
+  width: 30%;
+  border: 1px solid gray;
+  margin: 5px;
+}
+
+/* displays 2 cards in a row on table screens with room for margins */
+@media screen and (max-width: 768px) {
+  .dog-adoption-card {
+    width: 45%;
+    border: 1px solid gray;
+    margin: 5px;
+  }
+}
+
+/* displays 1 card in a row on mobile screens with room for margins */
+@media screen and (max-width: 576px) {
+  .dog-adoption-card {
+    width: 90%;
+    border: 1px solid gray;
+    margin: 5px;
+  }
+}
+
+.adopt-button {
+  background-color: #93B255;
+  font-size: 1rem;
+  border: none;
+  color: white;
+  padding: .5rem 1rem;
+  text-align: center;
+  text-decoration: none;
+  display: block;
+  font-weight: bold;
+  margin-top: .25rem;
+  margin-right: 2rem;
+  margin-bottom: .5rem;
+  margin-left: 2rem;
+  cursor: pointer;
+}
+
+.dog-card-image {
+  max-width: 100%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.dog-name {
+  margin: 1rem;
+}
+
+.dog-adoption-cost {
+  margin: 1rem;
+}
+
+.dog-description {
+  margin: 1rem;
+}
+
+/* home page styling */
+.intro {
+    margin: 0rem 5rem 4rem 1rem;
+    width: 100%;
+    overflow: auto;
+}
+.intro-image {
+    display: inline-block;
+    width: 50%;
+    float: left;
+    margin: 1rem 1rem .5rem 1rem;
+}
+
+.intro-content {
+    margin-left: 2rem;
+    margin-right: 2rem;
+    margin-top: 0px;
+}
+
+.featured-dogs-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-bottom: 1rem;
+}
+
+.main-content {
+  width: 66%;
+  float: left;
+  margin: 5px;
+}
+
+aside {
+  width: 30%;
+  float: right;
+  margin-right: 5px;
+}
+
+.show-more-button {
+  background-color: darkgray;
+  font-size: 1rem;
+  border: none;
+  color: #263E60;
+  padding: .5rem 1rem;
+  text-align: center;
+  text-decoration: none;
+  display: block;
+  font-weight: bold;
+  margin-top: .25rem;
+  margin-right: auto;
+  margin-bottom: .25rem;
+  margin-left: auto;
+  cursor: pointer;
+  width: 25%;
+}
+
+.blog-link {
+  color: #279cba;
+  font-weight: bold;
+}
+
+.featured-blog-entry {
+  margin-bottom: 2rem;
+}
+
+/* show blog previews only on iPad portrait orientation or higher */
+@media screen and (min-width: 0px) and (max-width: 768px) {
+  .featured-blog-image {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 0px) and (max-width: 768px) {
```

Comment:
You can move both of these into the same media query, no need to declare it twice.